src/store/index.js

```diff
@@ -8,31 +8,40 @@ Vue.use(Vuex);
 export default new Vuex.Store({
   state: {
     movies: [],
-    searchQuery: '',
+    searchQuery: "",
     movieDetails: {},
+    isLoading: false,
   },
   getters: {
     uniqueMovies: (state) => {
       let uniqueMovies = uniqby(state.movies.results, "id");
       return uniqueMovies;
     },
+    searchQuery: (state) => state.searchQuery,
+    isLoading: (state) => state.isLoading,
```

Comment:
This getter is unnecessary. Just use `mapState` directly if you need to fetch "isLoading" from the state.

Same goes for the searchQuery getter on line `20`.